docs/docs/intro.md

```diff
@@ -20,7 +20,7 @@ While Sedge assists in installing the validator, it is not designed to register
 
 :::caution Disclaimer
 
-Users acknowledge that no warranty is being made of a successful installation. Sedge is a tool and ultimately depends on you to use it correctly and following all the best practice guidance, as found in the project's README and this documentation.
+Users acknowledge that no warranty is being made of a successful installation. Sedge is a tool and it depends on you to use it correctly and follow all the best practice guidance, as found in the project's README and this documentation.
```

Comment:
Most of the disclaimers are in the README.md (at the root folder) as well, can you please propagate any disclaimer changes to the README?

Reply:
Yes, that's fine - I just checked Readme too.